include/jemalloc/internal/tsd.h

```diff
@@ -355,8 +355,8 @@ tsd_assert_fast(tsd_t *tsd) {
 	 * counters; it's not in general possible to ensure that they won't
 	 * change asynchronously from underneath us.
 	 */
-	assert(!malloc_slow && tsd_tcache_enabled_get(tsd) &&
-	    tsd_reentrancy_level_get(tsd) == 0);
+	assert(!malloc_initialized() || (tsd_tcache_enabled_get(tsd) &&
```

Comment:
Why do we have the additional `!malloc_initialized()` condition here?

Reply:
Let me see if I can get rid of it. I wanted to call this during tsd init, which might break the previous assertions.

Reply:
Got rid of it.